src/main/scala/nl/biopet/sbtbiopet/BiopetKeys.scala

```diff
@@ -0,0 +1,14 @@
+package nl.biopet.sbtbiopet
+
+import sbt._
+
+trait BiopetKeys {
```

Comment:
Is it required to have this in a separated file?

Reply:
Not necessarily, but it keeps things clean